### PR TITLE
Add error messaging to pivot DuplicateValueError failures

### PR DESCRIPTION
**Pull request by jmuharsky**
This resolves an issue where certain pivot errors were returning an empty string.

### CODE REVIEW

**Comment by jmuharsky** [comment]
Demo is at https://27-1-dot-perfkit-explorer.googleplex.com

File: server/perfkit/common/big_query_result_pivot.py [hunk 1]
```diff
@@ -173,10 +173,9 @@ def Transform(self, modify_reply=True, initialize_first=True):
 
       # If the target cell already has a value, throw an error.
       if self.transformed_rows[row_index]['f'][column_index]['v']:
-        logging.error(
-            'Pivot failed: value already exists at row "%s", col "%s".',
-            row_name, column_name)
-        raise DuplicateValueError()
+        msg = 'Pivot failed: value already exists at row "%s", col "%s".' % (row_name, column_name)
```
Comment (ivansmf) [inline server/perfkit/common/big_query_result_pivot.py]: Can you make the action the user should take more obvious? In this case the aggregate for the pivot does not produce unique values for the cell, and the typical error message is "Cannot mix aggregate and non-aggregate arguments". My suggestion would be "Cannot mix aggregate and non-aggregate arguments - found multiple values for row "%s", column "%s", but don't know how to aggregate. Please review your pivot definition."

**Comment by ivansmf** [comment]
I suggested a more actionable error message, could you please review?

**Comment by ivansmf** [comment]
Dear Googlebot, you're drunk, go home.
https://g3doc.corp.google.com/company/teams/opensource/cla/index.md?cl=head#googlers

**Comment by jmuharsky** [comment]
Good otherwise?  I updated the error message.

**Pull request merged**